examples/test-samples/project-invalid-sample.json

```diff
@@ -314,4 +314,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
We have `prettier` up and running before commits now. That's the reason for all these fixes. We used to have `pretty-quick` but the `prettier` needed for `pretty-quick` to run was missing before.